tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -331,15 +331,17 @@ test.describe( 'Nested Tabs tests', () => {
 		// Act.
 		// Add tab icons.
 		await setIconsToTabs( page, TabsIcons );
-		const activeTabSpanCount = await editor.getPreviewFrame().locator( '.e-normal.e-active span' ).count();
+
+		// Assert.
```

Comment:
This is only a test... Where is the fix?

Reply:
@udidol 
I only did a tweak to the existing test.